Swat/SwatFileEntry.php

```diff
@@ -283,21 +257,18 @@ public function getUniqueFileName($path)
     {
         if (is_dir($path)) {
             return $this->generateUniqueFileName($path);
-        } else {
-            throw new SwatException(
-                "Path '{$path}' is not a " . 'directory or does not exist.',
-            );
         }
-    }
 
-    // }}}
-    // {{{ public function getTempFileName()
+        throw new SwatException(
+            "Path '{$path}' is not a " . 'directory or does not exist.',
```

Comment:
This is some janky string concatenation. It can be one string.

Reply:
Agreed.